pkg/apis/extensions/validation/validation.go

```diff
@@ -698,6 +698,9 @@ func ValidatePodSecurityPolicySpecificAnnotations(annotations map[string]string,
 	}
 	if allowed := annotations[seccomp.AllowedProfilesAnnotationKey]; allowed != "" {
 		for _, p := range strings.Split(allowed, ",") {
+			if p == seccomp.AllowAny {
```

Comment:
nit: should this live in `ValidateSeccompProfile` where the other checks are?

Reply:
No, because `*` is not a valid profile, it is only valid in the context of allowed profiles.

Reply:
fair enough, the other values in there are interpreted but are actually passed back as actual profiles.